source/Ship.cpp

```diff
@@ -1078,6 +1082,27 @@ int64_t Ship::ChassisCost() const
 
 
 
+int64_t Ship::Strength() const
+{
+	return Cost() * Deterrence();
```

Comment:
Do we need to multiply cost here? It seems like (at least to me) that cost was used because there wasn't anything better. But now you can use deterrence to measure the strength of a ship. So multiplying by cost seems unnecessary.

Reply:
I didn't want to only use deterrence, I feel like thats not the only thing that should matter.

The cost of the ship is a good estimate of its hull and battle capabilities in general, paired with the ability to dish out damage that is deterrence.

Reply:
Also, as noted in my earlier comment, there are weaknesses to the deterrence cost, so I wouldn't suggest basing anything solely on that without first fixing it.